wrappers/kratos_CFD_wrapper.py

```diff
@@ -535,46 +561,23 @@ def initialize(self):
             self.fluid_model_part,
             self.SolverSettings)
 
-        pass
-
-    def initializeTimeStep(self):
-        pass
+        self.addNodalVariablesToModelpart(self.fluid_model_part)
 
     def run(self):
         """ Run a step of the wrapper """
 
-        newFluidMp = ModelPart("")
-
-        self.fluid_model_part = newFluidMp
-
-        self.__addNodalVariablesToModelpart(self.fluid_model_part)
-
-        # model_part_io_fluid = ModelPartIO("CFD_exampleFluid")
-        # model_part_io_fluid.ReadModelPart(self.fluid_model_part)
-
         # Import the into Kratos
-        self.__importKratosElements(
+        self.importKratosNodes(
             self.get_mesh("Model"),
-            newFluidMp
+            self.fluid_model_part
```

Comment:
only a mesh named "Model" is ever used?    It appears that if a user adds a mesh with another name (using ```add_mesh```), then that mesh is never used.